### PR TITLE
[issue] update a4 to fix poll and small style and wording fixes

### PR DESCRIPTION
**Pull request by phillimorland**

### CODE REVIEW

**Comment by Rineee** [comment]
@phillimorland The build fails, something wrong with dependencies?

**Comment by phillimorland** [comment]
@Rineee working, just needed a rebuild and a white space

**Comment by Rineee** [comment]
> @Rineee working, just needed a rebuild and a white space

@phillimorland Cool! Is it still WIP or ready to review?

**Comment by phillimorland** [comment]
@Rineee I just found another bug in a4 when testing the build so changed to WIP, sorry!

File: apps/interactiveevents/assets/QuestionUser.jsx [hunk 1]
```diff
@@ -74,8 +74,8 @@ export default class QuestionUser extends React.Component {
                     type="button"
                     className={'rating-button rating-up ' + (this.state.session_like ? 'is-selected' : '')}
                     onClick={this.handleLike.bind(this)}
-                    aria-label={this.state.session_like ? ariaAddLike : ariaUndoLike}
-                    title={this.state.session_like ? ariaAddLike : ariaUndoLike}
+                    aria-label={this.state.session_like ? ariaUndoLike : ariaAddLike}
```
Comment (fuzzylogic2000) [inline apps/interactiveevents/assets/QuestionUser.jsx:77]: Good catch!

**Review by fuzzylogic2000 — COMMENTED** [review]
The code looks good to me, but the past module tiles break a bit, the status bar is not over the whole thing, but has padding
Screenshot from 2023-01-25 14-32-01

**Comment by phillimorland** [comment]
damm, will take a look :)

**Comment by phillimorland** [comment]
@fuzzylogic2000 comments are there and the styling looks correct and seems to catch when I change things so hopefully fixed?

**Review by fuzzylogic2000 — APPROVED** [review]
Yes! It does look good now!

**Pull request merged**